src/gwt/src/org/rstudio/studio/client/workbench/views/connections/ui/NewConnectionInstallPackagePage.java

```diff
@@ -74,8 +81,22 @@ public void onBeforeActivate(Operation operation, final ModalDialogBase wizard)
                @Override
                public void execute()
                {
-                  wizard.closeDialog();
-                  connectionsPresenter_.onNewConnection();
+                  server_.connectionAddPackage(info_.getPackage(), new ServerRequestCallback<Void>()
+                  {
+                     @Override
+                     public void onResponseReceived(Void empty)
+                     {
+                        wizard.closeDialog();
+                        connectionsPresenter_.onNewConnection();
+                     }
+                     
+                     @Override
+                     public void onError(ServerError error)
+                     {
+                        Debug.logError(error);
```

Comment:
Should we show this error to the user? (seems like they'll just see the dialog disappear w/o any explanation)